docs/es/week03/03-1.md

```diff
@@ -368,18 +366,17 @@ Another type of neuron, which they named "complex cells", aggregate the output o
 -->
 
 
-### Contribuciones de Hubel & Weisel's (1962)
+### Contribuciones de Hubel y Wiesel (1962)
 
 <center><img src="{{site.baseurl}}/images/week03/03-1/Hubel_and_Weisel.png" alt="Hubel_and_Weisel" style="zoom:55%;" /></center>
 
-<div align="center">Fig. 13. Los experimentos de Hubel & Weisel mediante estímulos en cerebros de gatos <div>
-
-En sus experimentos, Hubel y Weisel utilizaron electrodos para medir activaciones neuronales en respuesta a estímulos visuales en cerebros de gatos. Con estos, descubrieron que las neuronas en la región V1 son sensibles únicamente a ciertas zonas de un campo visual (llamadas "campos receptores"), en este pueden detectar orillas con alguna orientación. Por ejemplo, los investigadores demostraron que si se le es enseñado una orilla con una orientación dada a un gato, y esta es rotada, a un ángulo dado una neurona se activará. Asímismo, cuando esta figura se mueve de este ángulo, la activación de la neurona decrece. A estas neuronas de activación selectiva Hubel y Weisel las llamaron "células simples", por su habilidad para detectar patrones locales.
+<div align="center">Fig. 13. Los experimentos de Hubel y Wiesel mediante estímulos en cerebros de gatos</div>
 
+En sus experimentos, Hubel y Wiesel utilizaron electrodos para medir activaciones neuronales en respuesta a estímulos visuales en cerebros de gatos. Con estos, descubrieron que las neuronas en la región V1 son sensibles únicamente a ciertas zonas de un campo visual (llamadas "campos receptores") y detectan bordes orientados. Por ejemplo, Hubel y Wiesel demostraron que si muestras a un gato una barra vertical y comienzas a rotarla, a un ángulo dado la neurona se activará. De manera similar, cuando esta barra se aleja de ese ángulo, la activación de la neurona decrece. A estas neuronas de activación selectiva Hubel y Wiesel las llamaron "células simples", por su habilidad para detectar patrones locales.
 
-De igual manera, descubrieron que si la barra es movida fuera del campo receptor, esa neurona en particular dejará de activarse, pero alguna otra lo hará. Existen detectores de patrones locales que corresponden a todas las áreas del campo visual, de ahí la idea de que  el cerebro humano procesa la información como una colección de "convoluciones".
+Ellos también descubrieron que si alejas la barra del campo receptor, esa neurona en particular dejará de activarse, pero alguna otra lo hará. Existen detectores de patrones locales que corresponden a todas las áreas del campo visual, y de ahí la idea de que  el cerebro humano procesa la información visual como una colección de "convoluciones".

```

Comment:
suggestion: "...**si alejas**..." to "...**si se aleja**..." (tense homogeneity)

Reply:
Agreed with the change. Although it is not the *tense* but the *voice*. The original uses impersonal *you*, I misread it as *you*. So I agree it's better to use the passive voice in Spanish.